apollo-adminservice/src/main/resources/application-mysql.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2023 Apollo Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+spring.datasource.hikari.connection-init-sql = SET NAMES utf8mb4
```

Comment:
we need to remove the `connection-init-sql` configuration in `application-github.properties`.